src/firefly/js/core/MasterSaga.js

```diff
@@ -30,6 +31,7 @@ export function* masterSaga() {
     // This section starts any predefined Sagas
     yield fork( watchCatalogs, null, dispatch, getState);
     yield fork( imagePlotter, null, dispatch, getState);
+    yield fork( syncCharts, null, dispatch, getState);
```

Comment:
Question for you and @robyww, is this where we are suppose to initialize these sagas?
What if it's not needed?